Increased color contrast when buttons overlap selected rows

jonbell-lot23 opened the pull request:
We have a long-standing issue where our blue time travel button disappears when it overlaps on a selected row. I've been playing around with ways to make this work better. Not just here, but how the colors can be applied across the system.

Current approach on left, proposed on right:
![image](https://github.com/replayio/devtools/assets/9154902/455e4caf-de52-49f2-bb61-6c21380a61ef)
This is a no-brainer.

![image](https://github.com/replayio/devtools/assets/9154902/a76f0c56-0556-45fd-841f-154dc74d7185)
And this is a nice standardization/cleanup in the library.

jasonLaster commented:
I like this better. Also winter what it would look like with the lighter blue row background and the primary blue button.

jonbell-lot23 commented:
Following up on discord comment from yesterday: yeah, I like the green too but I also think we can stick with blue for now and keep the green in our back pocket. It's coming (probably)! Doesn't need to be rushed out now though.

bvaughn commented:
I like this change but it only update some parts of our app, so it causes inconsistencies. For example the Network panel looks like this now
<img width="797" alt="Screen Shot 2023-11-09 at 4 46 13 PM" src="https://github.com/replayio/devtools/assets/29597/ae057793-dafa-4a10-b579-179f3ccaada2">

but the Elements and React panels still look like this
<img width="557" alt="Screen Shot 2023-11-09 at 4 47 20 PM" src="https://github.com/replayio/devtools/assets/29597/12b15671-c633-4ef9-b1c8-167915caf966">
<img width="559" alt="Screen Shot 2023-11-09 at 4 47 25 PM" src="https://github.com/replayio/devtools/assets/29597/af43f4d0-5925-4600-ba5d-92307ced96a5">

bvaughn commented on packages/replay-next/pages/variables.css:
```diff
@@ -709,7 +716,7 @@
   --theme-text-field-color: var(--body-color);
   --theme-text-field-placeholder-color: #a9a9b1;
-  --team-row-active: #3f434a;
+  --team-row-active: #163e58; /* the dark version of --primary-accent-dimmed */
```
btw
```suggestion
  --team-row-active: var(--primary-accent-dimmed);
```

jonbell-lot23 replied:
Usually variables are great. But in this case it's the left-hand column in the library, which always stays dark. Using the variable here means it uses the wrong color when in light theme.

bvaughn replied:
Oh I see. Hmm 🤔 That's pretty easy to get mixed up.

jonbell-lot23 replied:
Indeed. I run against this every. single. time.

bvaughn commented:
Noticed light theme has a contrast issue with the duration bubble
<img width="555" alt="Screen Shot 2023-11-09 at 4 50 33 PM" src="https://github.com/replayio/devtools/assets/29597/ec9d5d2c-526a-4c18-90ca-869f064849d8">

jonbell-lot23 commented:
Oh good -- if you think it works fine in those cases, I'll add them here. I was going to address that in a follow-up because I assumed the HTML colors would cause more issues and edge cases.

(Philosophically there are two ways to look at this -- find everything related and put it into a single PR versus stage things out, but we don't have to get into that here, happy to put this into one)

jonbell-lot23 commented:
@bvaughn I am having trouble figuring out what file changes the selected color in the react devtools. I tried deleting everything from variables.css but the blue still comes through, meaning it's not using variables. Then I looked in files like src/ui/components/SecondaryToolbox/react-devtools/components/ReactDevToolsListItem.module.css but I couldn't manage to find the right spot.

jonbell-lot23 commented:
> I like this change but it only update some parts of our app, so it causes inconsistencies. 

Updated these:

<img width="344" alt="image" src="https://github.com/replayio/devtools/assets/9154902/d8215833-0cff-4bf3-8b8d-d2a70f2533ea">

<img width="295" alt="image" src="https://github.com/replayio/devtools/assets/9154902/474dd646-66ea-4fd1-a5f2-43a5c6feb45d">

> Noticed light theme has a contrast issue with the duration bubble.

Updated these:

<img width="289" alt="image" src="https://github.com/replayio/devtools/assets/9154902/e6af5075-4ef5-457f-b186-9a4c4568125c">

<img width="289" alt="image" src="https://github.com/replayio/devtools/assets/9154902/a4b63481-6b51-4f92-ac03-3118d596e780">

ryanjduffy reviewed (approved):
:+1: Adding some snapshot tests for these changes would be nice too
